test/integration/scheduler_perf/scheduler_test.go

```diff
@@ -134,11 +133,10 @@ func schedulePods(config *testConfig) int32 {
 	// We are interested in low scheduling rates (i.e. qps=2),
 	minQPS := int32(math.MaxInt32)
 	start := time.Now()
-	podLister := config.schedulerSupport.PodInformer.Lister()
 	// Bake in time for the first pod scheduling event.
 	for {
 		time.Sleep(50 * time.Millisecond)
-		scheduled, err := getScheduledPods(podLister)
+		scheduled, err := getScheduledPods(config.clientset)
```

Comment:
Don't use clientset as it issues instant call to apiserver without reusing client-side caching.

Reply:
done